nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1989 @@
+            claimName: ${APPLICATION_NAME}-uat-binaries
+        - name: nuxeoconf
+          configMap:
+            name: ${APPLICATION_NAME}-uat-conf
+        - name: connect-secret
+          secret:
+            secretName: ${APPLICATION_NAME}-connect
+        - name: nuxeodata
+          emptyDir: {}
+        - name: nuxeolog
+          emptyDir: {}
+        - name: nuxeotmp
+          emptyDir: {}
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        terminationGracePeriodSeconds: 30
+    test: false
+    triggers:
+    - type: ConfigChange
+    - type: "ImageChange"
+      imageChangeParams:
+        automatic: false
+        containerNames:
+        - ${APPLICATION_NAME}-appserver
+        from:
+          kind: ImageStreamTag
+          name: ${APPLICATION_NAME}-nuxeo:latest
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: ${APPLICATION_NAME}-uat-nuxeo
+    labels:
+      app: ${APPLICATION_NAME}-uat
+      component: nuxeo
+      role: lb
+  spec:
+    clusterIP: None
+    selector:
+      app: ${APPLICATION_NAME}-uat
+      component: nuxeo
+      stage: uat
+    ports:
+    - name: 8080-tcp
+      port: 8080
+      protocol: TCP
+
+- apiVersion: v1
+  kind: Route
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}-uat
+    name: ${APPLICATION_NAME}-uat-nuxeo-route
+  spec:
+    # TODO: Add parameter
+    host: ${APPLICATION_NAME}-uat.${DOMAIN_SUFFIX}
+    tls:
+      termination: edge
+      insecureEdgeTerminationPolicy: Redirect
+    to:
+      kind: Service
+      name: ${APPLICATION_NAME}-uat-nuxeo
+    port:
+      targetPort: 8080-tcp
+    wildcardPolicy: None
+
+
+
+############################################################################################################
+# Build pipeline
+############################################################################################################
+
+
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: ${APPLICATION_NAME}-base-image
+    labels:
+      app: ${APPLICATION_NAME}
+  spec: {}
+  status:
+    dockerImageRepository: ""
+
+
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+      component: build
+    name: ${APPLICATION_NAME}-base-image-build
+  spec:
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: '${APPLICATION_NAME}-base-image:latest'
+    postCommit: {}
+    resources: {}
+    runPolicy: Serial
+    source:
+      dockerfile: |-
+          FROM nuxeo:${NUXEO_VERSION}
+          USER root
+
+          # Reinstall ImageMagick with the RSVG delegate
+          RUN apt-get update && \
+              apt-get remove -y imagemagick && \
+              apt-get install -y --no-install-recommends librsvg2-bin && \
+              apt-get install -y imagemagick && \
+              apt-get install -y --no-install-recommends vim && \
+              wget http://apt.nuxeo.org/pool/releases/ffmpeg-nuxeo_2.8.5-1_amd64.deb && \
```

Comment:
`ffmpeg` is now part of the Docker image, you can safely remove its installation.

Reply:
removed